rover_msgs/include/rovus_lib/macros.h

```diff
@@ -49,5 +49,8 @@
 
 // Removes unused argument warning 
 #define REMOVE_UNUSED(x) (void)(x)
+
+#define GET_PACKAGE_SOURCE_DIR(package_name) \
+    (ament_index_cpp::get_package_prefix(package_name) + "/../../src/rover/" + package_name)
```

Comment:
Consider adding the necessary ament_index_cpp #include at the top of the file. Otherwise people using the macro will need to include it in their projects which defeats the purpose